doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1773,18 +2124,11 @@ performance issue.
    and allow displaying and resetting the profile from tactic scripts for
    benchmarking purposes.
 
-.. tacn:: show ltac profile
+.. tacn:: show ltac profile {? {| cutoff @int | @string } }
    :name: show ltac profile
 
-   This tactic behaves like the corresponding vernacular command
-   and allow displaying and resetting the profile from tactic scripts for
-   benchmarking purposes.
-
-.. tacn:: show ltac profile @string
-   :name: show ltac profile
-
-   This tactic behaves like the corresponding vernacular command
-   and allow displaying and resetting the profile from tactic scripts for
+   Behaves like the corresponding vernacular command :cmd:`Show Ltac Profile`,
```

Comment:
Let's reduce a bit the use of "vernacular" in the doc:
```suggestion
   Behaves like the corresponding command :cmd:`Show Ltac Profile`,
```

Note that the `reset ltac profile` above could be similarly adapted to link to the corresponding command.